riposte/riposte.py

```diff
@@ -18,10 +18,12 @@ class Riposte(PrinterMixin):
     def __init__(
         self,
         prompt: str = "riposte:~ $ ",
+        splash: str = None,
```

Comment:
```suggestion
        splash: Optional[str] = None,
```

Reply:
Also, I think that `banner` is more appropriate than `splash`.

Reply:
Happy to make this change